iree/base/logging.cc

```diff
@@ -108,7 +108,7 @@ void LogMessage::EmitLogMessage() {
   {
     // NOTE: this truncates. That's fine for now and stderr is still usable.
     char str_buffer[512];
-    int str_length = snprintf(str_buffer, sizeof(str_buffer), "%s:%d] %s\n",
+    snprintf(str_buffer, sizeof(str_buffer), "%s:%d] %s\n",
                               file_name_, line_, str().c_str());
```

Comment:
run `clang-format` (since the unused variable was dropped)

Reply:
I sent a cl. Should i just add a commit here instead?

Reply:
Either is fine.